src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-details/rgw-storage-class-details.component.html

```diff
@@ -108,9 +106,20 @@
             <td>{{ selection?.multipart_sync_threshold }}</td>
           </tr>
           <tr>
-            <td
-                class="bold">
-              Retain Head Object
+            <td class="bold">
+              <span i18n>Allow Read Through </span>
+              <cd-helper class="text-pre-wrap">
+                <span i18n>
+                  Enables fetching objects from remote cloud S3 if not found locally (default:
+                  false).
+                </span>
+              </cd-helper>
+            </td>
+            <td>{{ selection?.allow_read_through }}</td>
+          </tr>
+          <tr>
+            <td class="bold">
+              Head Object (Stub File)
```

Comment:
```suggestion
              Head Object 
```

why stub file ? that seems very technical and confusing  no ?

Reply:
Suggested by @likid0 !